test/index.ts

```diff
@@ -43,17 +47,20 @@ const fakeGoogleGax = {
 };
 
 const subscriptionCached = subby.Subscription;
-let subscriptionOverride;
+// tslint:disable-next-line no-any
+let subscriptionOverride: any;
```

Comment:
can you please change this to like they suggested in other PR?

Reply:
done